packages/frontend/src/view/pages/user/components/UserAssetTable.tsx

```diff
@@ -34,6 +35,15 @@ export function UserAssetsTable(props: UserAssetsTableProps) {
         }/${entry.asset.hashOrId.toString()}`
       : `/forced/new/${entry.vaultOrPositionId}`
 
+  const escapeHatchElem = (entry: UserAssetEntry) =>
+    entry.action === 'WITHDRAW' ? (
+      <LinkButton href={`/escape/${entry.vaultOrPositionId}`}>
+        ESCAPE
+      </LinkButton>
+    ) : (
+      <span className="text-zinc-500">use collateral escape</span>
```

Comment:
That would require improvement

Reply:
I'll discuss this with Janek (as all the other UIs I did myself)